### ISSUE TITLE
Incorrect arity 5 for io.pedestal.http.route.definition.table/syntax-error

### ISSUE THREAD

**Issue by souenzzo** [pedestal/pedestal#683]
# Description

At `io.pedestal.http.route.definition.table/make-parse-context` there is an `asset` that calls `syntax-error` in case of `failure`

## Expected Behavior

```
(make-parse-context nil nil nil)
Execution error (AssertionError) at io.pedestal.http.route.definition.table/make-parse-context (table.clj:38).
...
```

## Actual Behavior

```
(make-parse-context nil nil nil)
Execution error (ArityException) at io.pedestal.http.route.definition.table/make-parse-context (table.clj:38).
Wrong number of args (5) passed to: io.pedestal.http.route.definition.table/syntax-error 
```

## Steps to reproduce

`(make-parse-context nil nil nil)`

# Environment

## Pedestal version
 0.5.8

**Comment by ddeaguiar** [comment]
Thanks for reporting @souenzzo!

**Comment by ddeaguiar** [comment]
Fixed in [97b3443426](https://github.com/pedestal/pedestal/commit/97b3443426cbd67393a9725fa95c8515e5d2b0c3).
